src/main/java/com/liferay/faces/bridge/ext/filter/internal/BridgePortletConfigFactoryLiferayImpl.java

```diff
@@ -41,22 +41,23 @@ public BridgePortletConfigFactoryLiferayImpl(BridgePortletConfigFactory bridgePo
 	@Override
 	public PortletConfig getPortletConfig(PortletConfig portletConfig) {
 
+		PortletConfig savedPortletConfig = this.portletConfig;
```

Comment:
@ngriffin7a, I made the double-checked locking pattern here more consistent with [the pattern used in `GenericFacesPortlet`](https://github.com/liferay/liferay-faces-bridge-api/blob/master/bridge-api/src/main/java/javax/portlet/faces/GenericFacesPortlet.java#L770-L789).